main.go

```diff
@@ -4,10 +4,37 @@ import (
 	"fmt"
 )
 
+var unusedVar int
```

Comment:
Remove the `unusedVar` variable if it's not needed. This will improve code readability and maintainability. [medium]